Dockerfile

```diff
@@ -1,10 +1,4 @@
-FROM ubuntu
-RUN echo "deb http://archive.ubuntu.com/ubuntu precise universe" >> /etc/apt/sources.list
-RUN apt-get update
-RUN apt-get install -y software-properties-common
-RUN apt-get install -y python-software-properties python g++ make
-RUN add-apt-repository -y ppa:chris-lea/node.js
-RUN apt-get update
-RUN apt-get install -y nodejs
+FROM node
```

Comment:
please use a tagged revision there, like node:0.10. Node version 0.12 is coming out at some point, and I do not want to touch those again